docs/tutorials/protect_against_regen.rst

```diff
@@ -0,0 +1,47 @@
+.. include:: ../text_colors.rst
+.. toctree::
+
+.. _protect_against_regen:
+
+*************************************************************************
+Configuring Your Module to Protect Against Regeneration Voltage Spikes
+*************************************************************************
+
+What are Regenerative Voltage Spikes
+=======================================
+Regeneration voltage occurs when the commanded module voltage is less than the module's back-EMF voltage. When this occurs, the motor's kinetic 
```

Comment:
What sort of customer situations does this happen in? Our customers are often not so in the know about actual motor physics stuff like Back-EMF and all. So it may be good to give some ideas here of when this would happen in terms of what they think of. E.g. When the motor is slowing down or braking. And a harder step down will generally produce a more extreme current.